### PR TITLE
Updated to use Swift 5 Result type.

### PR DESCRIPTION
**Pull request by michael-patzer**
Added extension to Result to decode JSON. Updated API paths to remove double slash. Updated to use The Rocket Science Group LLC development team.

### CODE REVIEW

File: Anzee/Requests/CampaignRequest.swift [hunk 1]
```diff
@@ -13,7 +13,7 @@ struct CampaignRequest: APIRequest {
 
     var params: [URLQueryItem]?
 
-    var apiPath: String { return "/campaigns/\(campaignID)" }
+    var apiPath: String { return "campaigns/\(campaignID)" }
 
     public typealias CampaignRequestCallback = ((_ campaign: Campaign?, _ error: APIError?) -> Void)
```
Comment (shawn-veader) [inline Anzee/Requests/CampaignRequest.swift]: @michael-patzer Would it be too much to have these use a `Result` type too?

**Review by shawn-veader — APPROVED** [review]

File: Anzee/Requests/CampaignRequest.swift (hunk 1 above)
Comment (protocolharbor) [reply Anzee/Requests/CampaignRequest.swift]: @veader Good idea. PR updated.

File: Anzee.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -19,6 +19,7 @@
 		76CE62B3225466D0007376BB /* APIRequest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 76CE62B2225466D0007376BB /* APIRequest.swift */; };
 		76CE62B522546706007376BB /* AnzeeAPI.swift in Sources */ = {isa = PBXBuildFile; fileRef = 76CE62B422546706007376BB /* AnzeeAPI.swift */; };
 		76CE62B822546827007376BB /* CampaignsRequest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 76CE62B722546827007376BB /* CampaignsRequest.swift */; };
+		B95896C42278D93500F7509B /* Result.swift in Sources */ = {isa = PBXBuildFile; fileRef = B95896C32278D93500F7509B /* Result.swift */; };
```
Comment (veader) [inline Anzee.xcodeproj/project.pbxproj:22]: I could have sworn I saw this previously... I don't see this actual file in the repo.

Comment (protocolharbor) [reply Anzee.xcodeproj/project.pbxproj:22]: My mistake. Added to git.

**Review by veader — APPROVED** [review]
👍  Looks great.

**Pull request merged**